distribution/index.js

```diff
@@ -170,12 +170,6 @@ const execFile = util.promisify(__webpack_require__(129).execFile);
 
```

Comment:
`distribution` should be have the [`linguist-generated`](https://github.com/github/linguist#generated-code) git attribute so its diffs are collapsed